Functions.Templates/Templates/DurableFunctionsEntityHttpStart-CSharp-2.x/DurableFunctionsEntityHttpStartCSharp.csx

```diff
@@ -0,0 +1,22 @@
+#r "Microsoft.Azure.WebJobs.Extensions.DurableTask"
+#r "Newtonsoft.Json"
+
+using System.Net;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+
+public static async Task<HttpResponseMessage> Run(
+    [HttpTrigger(AuthorizationLevel.Function, Route = "Counter/{entityKey}")] HttpRequestMessage req,
+    [DurableClient] IDurableEntityClient client,
```

Comment:
This was throw compile error. The attributes are not needed in csx file.
```c#
[HttpTrigger(AuthorizationLevel.Function, Route = "Counter/{entityKey}")] HttpRequestMessage req, [DurableClient] IDurableEntityClient client
```
Was this tested locally or on portal?